src/index.js

```diff
@@ -13,6 +13,36 @@ const runSql = util.promisify(connection.query).bind(connection)
 const app = express()
 const port = config.PORT
 
+
+const MAX_ACTIVE = 2;
```

Comment:
could probably be a config option. i think 2 is quite low, i would probably set it to like 5 maybe. :thinking: